core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5156,6 +5156,9 @@ components:
         cloudPlatform:
           type: string
           description: The cloud platform of the workspace
+          enum: 
```

Comment:
I'm not sure whether this should be under "required" or not… I was going off the fact that we always now include this information in the response. @marctalbott, how do folks decide what to put under "required"?

Reply:
Ah never mind, I saw @marctalbott's response in the other PR.